{"body": "CircleCI is failing due to some package hash mismatch:\r\n\r\n```\r\nTHESE PACKAGES DO NOT MATCH THE HASHES FROM THE REQUIREMENTS FILE. If you have updated the package versions, please update the hashes. Otherwise, examine the package contents carefully; someone may have tampered with them.\r\n    click==6.6 from https://pypi.python.org/packages/1c/7c/10b4132dd952b6a04e37626258825b8aa8c1eb99545f2eb26a77c21efb55/click-6.6-py2.py3-none-any.whl#md5=ded5bf44698fd7def254d5cb9e5ad0b3 (from -c requirements/constraints.txt (line 13)):\r\n        Expected sha256 cc6a19da8ebff6e7074f731447ef7e112bd23adf3de5c597cf9989f2fd8defe9\r\n             Got        fcf697e1fd4b567d817c69dab10a4035937fe6af175c05fd6806b69f74cbc6c4\r\n```\r\n\r\nI believe @mythmon is working on a bug right now related to the build process", "body_text": "CircleCI is failing due to some package hash mismatch:\nTHESE PACKAGES DO NOT MATCH THE HASHES FROM THE REQUIREMENTS FILE. If you have updated the package versions, please update the hashes. Otherwise, examine the package contents carefully; someone may have tampered with them.\n    click==6.6 from https://pypi.python.org/packages/1c/7c/10b4132dd952b6a04e37626258825b8aa8c1eb99545f2eb26a77c21efb55/click-6.6-py2.py3-none-any.whl#md5=ded5bf44698fd7def254d5cb9e5ad0b3 (from -c requirements/constraints.txt (line 13)):\n        Expected sha256 cc6a19da8ebff6e7074f731447ef7e112bd23adf3de5c597cf9989f2fd8defe9\n             Got        fcf697e1fd4b567d817c69dab10a4035937fe6af175c05fd6806b69f74cbc6c4\n\nI believe @mythmon is working on a bug right now related to the build process", "url": "https://api.github.com/repos/mozilla/normandy/issues/comments/262049766", "created_at": "2016-11-21T19:59:10Z", "author_association": "CONTRIBUTOR", "html_url": "https://github.com/mozilla/normandy/pull/357#issuecomment-262049766", "updated_at": "2016-11-21T19:59:10Z", "user": {"following_url": "https://api.github.com/users/andymikulski/following{/other_user}", "events_url": "https://api.github.com/users/andymikulski/events{/privacy}", "organizations_url": "https://api.github.com/users/andymikulski/orgs", "url": "https://api.github.com/users/andymikulski", "gists_url": "https://api.github.com/users/andymikulski/gists{/gist_id}", "html_url": "https://github.com/andymikulski", "subscriptions_url": "https://api.github.com/users/andymikulski/subscriptions", "avatar_url": "https://avatars1.githubusercontent.com/u/2767162?v=4", "repos_url": "https://api.github.com/users/andymikulski/repos", "received_events_url": "https://api.github.com/users/andymikulski/received_events", "gravatar_id": "", "starred_url": "https://api.github.com/users/andymikulski/starred{/owner}{/repo}", "site_admin": false, "login": "andymikulski", "type": "User", "id": 2767162, "followers_url": "https://api.github.com/users/andymikulski/followers"}, "body_html": "<p>CircleCI is failing due to some package hash mismatch:</p>\n<pre><code>THESE PACKAGES DO NOT MATCH THE HASHES FROM THE REQUIREMENTS FILE. If you have updated the package versions, please update the hashes. Otherwise, examine the package contents carefully; someone may have tampered with them.\n    click==6.6 from https://pypi.python.org/packages/1c/7c/10b4132dd952b6a04e37626258825b8aa8c1eb99545f2eb26a77c21efb55/click-6.6-py2.py3-none-any.whl#md5=ded5bf44698fd7def254d5cb9e5ad0b3 (from -c requirements/constraints.txt (line 13)):\n        Expected sha256 cc6a19da8ebff6e7074f731447ef7e112bd23adf3de5c597cf9989f2fd8defe9\n             Got        fcf697e1fd4b567d817c69dab10a4035937fe6af175c05fd6806b69f74cbc6c4\n</code></pre>\n<p>I believe <a class=\"user-mention\" href=\"https://github.com/mythmon\">@mythmon</a> is working on a bug right now related to the build process</p>", "id": 262049766, "issue_url": "https://api.github.com/repos/mozilla/normandy/issues/357"}